config/application.rb

```diff
@@ -8,13 +8,16 @@
 require "action_view/railtie"
 require "sprockets/railtie"
 require "rails/test_unit/railtie"
+require "csv"
```

Comment:
Where do you need this? Wouldn't it be better to place it directly in the file you need it.